### ISSUE TITLE
Improve Accessiblity of Subscription Input and Button

### ISSUE THREAD

**Issue by machineloop** [vendure-ecommerce/storefront-qwik-starter#21]
Lighthouse A11y score catches the following issue:
<img width="711" alt="Screen Shot 2022-12-01 at 7 49 36 PM" src="https://user-images.githubusercontent.com/3682072/205189812-91cf1f43-14f8-4a81-ba8e-6676c31c76db.png">

1. Input is not properly labelled, though an invisible label is present for screenreaders, it's not associated with the Input. Adding an`id` to the label and `aria-labelledby` attribute to the input element should fix.
2. The Subscribe button has a poor contrast ratio of 3.68:1, doesn't meet [WCAG AA standard of 4.5:1](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast):
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=3482F6
Solution here would be to take the hover state which has darker background and higher contrast ratio of 5.16:1 and use it as the normal button color:https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=2563EB, then replace the hover state with a darker color like rgb(48, 48, 133). However these look like colors coming from tailwind, so may not be that straightforward.

**Comment by michaelbromley** [comment]
1. Yes this sounds fine
2. Correct - the colours come from Tailwind. To use the next darkest shade, we'd replace `bg-primary-500` with `bg-primary-600`, and change the hover class to `hover:bg-primary-700`.

A more maintainable solution would be to define a `btn-primary` class like we do here: 

https://github.com/vendure-ecommerce/storefront-angular-starter/blob/7e72017352532be11f49ede6290edc34e1bc8b85/src/styles/styles.scss#L6-L14

Then we could apply `btn-primary` to every primary CTA button and not have to duplicate all the classes.
